install/linux/usr/share/odemis/sim/meteor-sim.odm.yaml

```diff
@@ -71,14 +71,17 @@
         FM_IMAGING_RANGE: {"x": [0.040, 0.054], "y": [-5.e-3, 10.e-3], "z": [-0.5e-3, 8.e-3]},
         # Grid centers in SEM range
         # Adjusted so that at init (0,0,0), it's at the Grid 1.
-        SAMPLE_CENTERS: {"GRID 1": {'x': 0, 'y': 0, 'z': 0}, "GRID 2": {'x': 2.98e-3, 'y': 2.46e-3, 'z': 0}},
+        SAMPLE_CENTERS: {"GRID 1": {'x': 0, 'y': 0, 'z': 0}, "GRID 2": {'x': 5.0e-3, 'y': 0, 'z': 0}},
         # Mirroring values between SEM - METEOR
-        POS_COR: [0.02447, -0.000017],
+        POS_COR: [0.02447, -0.000017],  # [m] rotation correction (x, y) (sem <-> meteor)
+        POS_TRANS_COR: [48.8e-3, 0],    # [m] translation correction (x, y) (fib <-> meteor), standard value is x=48.8mm
```

Comment:
Given the comments, I am not sure if the metadata should be used like this. @pieleric can give a better answer

Reply:
I'm not sure I understand? The metadata is already being used for this, this only adds another parameter